spec/album_spec.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'rspec_helper'
+require_relative 'spec_helper'
```

Comment:
- [ ] Your Project is perfect. However, there are some gaps and changes that need to be done to approve this project.

You have three failing tests: 
<img width="776" alt="Screen Shot 2022-08-18 at 23 48 23" src="https://user-images.githubusercontent.com/69011963/185471358-23ba832b-f7ae-4bb4-964a-e6358acd24a5.png">

Kindly ensure that all tests are passing!